e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep

```diff
@@ -260,7 +260,7 @@ resource iotHub 'Microsoft.Devices/IotHubs@2020-01-01' = {
   sku: {
     name: 'S1'
     tier: 'Standard'
-    capacity: 1
+    capacity: 3
```

Comment:
our test suite requires the hub instance to have >1 unit, else our tests end up getting throttled or cancelled. We use 3 units for our canary instance so I used the same here.

Reply:
We had it at 1 for devs running tests against their own instance, and not incurring too much cost. As we do not deploy new pipeline resources that often, the idea was that you'd have to go manually change it on the pipeline's instance.

Reply:
I see, I'll add a flag to make this configurable and add a comment explaining this.

Reply:
As discussed offline, you can make this an ARM template parameter with a default of 1.